roles/ansible-core-ci-session/tasks/main.yaml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Ensure create_core_ci_session_key is set
+  assert:
+    that:
+      - create_core_ci_session_key is defined
+      - create_core_ci_session_key|length > 0
```

Comment:
I am concerned this could leak your key, adding no_log should help

Reply:
Ok